src/main/external-resources/renderers/Panasonic-DMRBWT740.conf

```diff
@@ -0,0 +1,56 @@
+#----------------------------------------------------------------------------
+# Profile for Panasonic DMR-BWT740.
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Panasonic BWT740
```

Comment:
With that line being changed we should also change the filename and the other occurrences of "DMR-BWT740" in this file. Otherwise we could make two files: `Panasonic-DMRBWT740.conf` and `Panasonic-DMRBWT.conf`.
They can be identical for now with their configs but it can set it up for future changes. What do you think?